reference/docs-conceptual/developer/cmdlet/how-to-invoke-a-cmdlet-from-within-a-cmdlet.md

```diff
@@ -74,12 +75,7 @@ namespace SendGreeting
   {
     // Declare the parameters for the cmdlet.
     [Parameter(Mandatory = true)]
-    public string Name
-    {
-      get { return name; }
-      set { name = value; }
-    }
-    private string name;
+    public string Name { get; set; }
```

Comment:
This is just a slight modernization of the example. Currently it would be a bit jarring for newer C# devs that weren't around when auto properties didn't exist.